sale_force_invoiced_qty/models/sale_order_line.py

```diff
@@ -0,0 +1,33 @@
+# Copyright (C) 2024 Cetmix OÜ
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+from odoo import _, api, fields, models
+from odoo.exceptions import UserError
+
+
+class SaleOrderLine(models.Model):
+    _inherit = "sale.order.line"
+
+    force_invoiced = fields.Float()
```

Comment:
You should use [precision](https://github.com/odoo/odoo/blob/fd67e6d683b05b84a0d30785c2ff3209d2c91cc3/addons/sale/models/sale_order_line.py#L223) here as well in order to match other quantity fields settings.